clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -57,6 +60,23 @@ func (ste *simpleTestExecutor) ExecuteTest(ctx Context, conf *api.Config) []erro
 			}
 		}
 	}
+
+	for _, summary := range ctx.GetMeasurementManager().GetSummaries() {
+		summaryText, err := summary.PrintSummary()
+		if err != nil {
+			errList = append(errList, fmt.Errorf("printing summary %s error: %v", summary.SummaryName(), err))
+			continue
+		}
+		if ctx.GetClusterConfig().ReportDir == "" {
+			glog.Infof("%v: %v", summary.SummaryName(), summaryText)
+		} else {
+			filePath := path.Join(ctx.GetClusterConfig().ReportDir, summary.SummaryName()+"_"+conf.Name+"_"+time.Now().Format(time.RFC3339)+".txt")
```

Comment:
Btw - one thing to keep in mind is to make this naming exactly same as it is currently (to maintain backward compatibility, for e.g it's needed for perf-dash).

Reply:
Naming format of file is the same, but I need to verify whether metrics have the same names.

Reply:
It looks like the structure of api latency summary is little different from original version. I will fix it in separate PR.